app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ProfilesController < ApplicationController
```

Comment:
Agreed. I had it stuck in my head that all Controllers in Rails are plural, but in this case a singular Profile Controller makes more sense.